noted/accounts/v1/groups.proto

```diff
@@ -58,3 +62,19 @@ message ListGroupMembersRequest {
 message ListGroupMembersResponse {
     repeated GroupMember members = 1;
 }
+
+message JoinGroupRequest {
+}
+
+message JoinGroupResponse {
+}
+
+message AddNoteToGroupRequest {
+    string id = 1;
+    string note_id = 2;
+}
+
+message AddNoteToGroupResponse {
+    string id = 1;
+    string member_id = 2;
+}
```

Comment:
I don't think we should return anything here. The reason why it's considered good practice to return data on create/update operations in REST APIs is to return the latest representation of the object within the API. This is useful when there has been side effect changes or when a consumer of the API needs to update the state of the object. Here the API object is a group so either we should return a `Group group = 1;` or we should return nothing. That's just my opinion so LMKWYT.

Reply:
j'ai suppr la réponse, pour le moment on à pas besoins de return le `Group` donc j'ai laissé blank